dj_rest_auth/urls.py

```diff
@@ -8,7 +8,7 @@
 urlpatterns = [
     # URLs that do not require a session or valid token
     path('password/reset/', PasswordResetView.as_view(), name='rest_password_reset'),
-    path('password/reset/confirm/', PasswordResetConfirmView.as_view(), name='rest_password_reset_confirm'),
+    path('password/reset/confirm/<uidb64>/<token>/', PasswordResetConfirmView.as_view(), name='password_reset_confirm'),
```

Comment:
this should be as is, due to we still need to be able to rest password via rest api
better solution, as for me, is to add `path('password/reset/<uidb64>/<token>/', PasswordResetConfirmView.as_view(), name='rest_password_reset_confirm_email'),`
and use in email reset template `rest_password_reset_confirm_email`
maintainers probably miss this, but so clues present in [demo app](https://github.com/iMerica/dj-rest-auth/blob/master/demo/demo/urls.py#L41)